packages/sanity/src/core/search/common/deriveSearchWeightsFromType.ts

```diff
@@ -19,7 +20,6 @@ const BASE_WEIGHTS: Record<string, Omit<SearchWeightEntry, 'path'>> = {
   _id: {weight: 1, type: 'string'},
   _type: {weight: 1, type: 'string'},
 }
-const GROQ_RESERVED_KEYWORDS = ['true', 'false', 'null']
```

Comment:
This doesn't seems to be handled by the `toString` function, do we need to add something else as safeguard?